utilities/mobile_server_pool.py

```diff
@@ -115,9 +165,12 @@ def cleanup_for_blocked_nodes():
         job_id = doc["username"]
         if job_id == "":
             job_id = "test_job:123"
-        job_name, build_id = job_id.split(":")
+        length_array = []
+        length_array = job_id.split(":")
+        job_name = length_array[0]
+        build_id = length_array[1]
         log_info("Releasing node {} for job {}".format(doc_id, job_id))
-        if not is_jenkins_job_running(job_name, build_id):
+        if job_name is not None or not is_jenkins_job_running(job_name, build_id):
```

Comment:
I am ok with this change, just thinking of how to prevent cleaning a node which is still running a jenkins job? any thought?

Reply:
we clean up in two different ways, 
one is release nodes -> this method used at post build actions once jenkins job is successful
second one is clean up blocked nodes -> This we have to run once in a while to clean up all the blocked nodes to available. if for any reason release nodes (above method) does not release nodes( could be jenkins down, power outages, unexpected job aborted), it leaves the nodes 'blocked' though jenkins jobs not running. Clean_up_blocked_nodes : clean up all blocked nodes if we notice nodes are occupied unnecessarily, but at the same time we should not interrupt if any of the jobs running.

Reply:
make sense, sounds good to me.